src/components/Home.jsx

```diff
@@ -0,0 +1,36 @@
+import { useSelector } from 'react-redux';
+import RecipeCard from './RecipeCard';
+import recipeLogo from '../assets/img/recipe.svg';
+
+const Home = () => {
+  const { recipes, isLoading } = useSelector((store) => store.recipes);
+
+  if (isLoading) {
+    return (
+      <div className="loading">
+        <p>Loading recipes...</p>
+      </div>
+    );
+  }
+
+  return (
+    <div className="home">
+      <div className="headline">
+        <img className="logo" src={recipeLogo} alt="logo" />
+        <p className="headline-text">Here is a list of the 20 best recipes for you to enjoy</p>
+      </div>
+      <div className="divider">Number of ingredients required</div>
+      {recipes.map((recipe) => (
+        <RecipeCard
+          key={recipe.key}
+          id={recipe.key}
+          name={recipe.label}
+          image={recipe.images.SMALL.url}
+          ingredientsCount={recipe.ingredients.length}
+        />
+      ))}
+    </div>
+  );
+};
```

Comment:
- [ ] It would be better to add a way to filter the recipes by name  I suggested adding input and filtering the recipe by name so that you complete the requirements 

- screenshot of the requirements 
<img width="991" alt="Screenshot 2023-04-19 at 22 34 02" src="https://user-images.githubusercontent.com/61790587/233214510-f601e210-dad2-4249-a3a3-c9090be46e24.png">

Reply:
Not resolved ✖️

Reply:
Hi @jlvFlores The search bar had to be visible on the homepage which wasn't at the time of my review...